lib/rules/property-no-deprecated/README.md

```diff
@@ -0,0 +1,86 @@
+# property-no-deprecated
+
+Disallow deprecated properties.
+
+<!-- prettier-ignore -->
+```css
+  a { word-wrap: break-word; }
+/**      ↑
+ * Deprecated property */
+```
+
+This rule flags properties that were removed or deprecated after being in the CSS specifications, including editor drafts, and were either:
+
+- shipped in a stable version of a browser
+- shipped by a developer channel/edition browser
+- shipped but behind experimental flags
+- polyfilled with some adoption before any browser actually shipped
+- had an MDN page at one point in time
+
+The [`fix` option](../../../docs/user-guide/options.md#fix) can automatically fix some of the problems reported by this rule.
+
+Prior art:
+
+- [@isnotdefined/no-obsolete](https://www.npmjs.com/package/@isnotdefined/stylelint-plugin)
+
+## Options
+
+### `true`
+
+```json
+{
+  "property-no-deprecated": true
+}
+```
+
+The following patterns are considered problems:
+
+<!-- prettier-ignore -->
+```css
+a { clip: rect(0, 0, 0, 0); }
+```
+
+<!-- prettier-ignore -->
+```css
+a { word-wrap: break-word; }
+```
+
+The following patterns are _not_ considered problems:
+
+<!-- prettier-ignore -->
+```css
+a { clip-path: rect(0, 0, 0, 0); }
```

Comment:
This is invalid CSS. As per https://developer.mozilla.org/en-US/docs/Web/CSS/basic-shape/rect, the correct syntax is without the commas:

```css
a { clip-path: rect(0 0 0 0); }
```

Reply:
> [Note: The rect() <shape> function used in the deprecated clip property is different from the CSS rect() function used to define a CSS <basic-shape>.](https://developer.mozilla.org/en-US/docs/Web/CSS/clip#values)

Indeed this

https://github.com/stylelint/stylelint/blob/c56de6a1a1b2e28fa17b25149fc644ba34925dd2/lib/rules/property-no-deprecated/index.mjs#L73

needs to be `null`.

Can you open an issue for it?